templates/devspace.yaml.tpl

```diff
@@ -239,6 +239,19 @@ dev:
           mountPath: /home/dev/.asdf/installs
           name: devspace-asdfcache
 
+      # asdf shims.
+      - op: add
```

Comment:
Why not reused the `asdfcache` PVC and instead use [`subPath`](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath) on each of these?